lab_5_scraper/scraper.py

```diff
@@ -189,6 +401,22 @@ def unify_date_format(self, date_str: str) -> datetime.datetime:
         Returns:
             datetime.datetime: Datetime object
         """
+        try:
```

Comment:
why are you parsing everything with try except? Try to avoid them or minimize the code in these statements